ground/src/main/java/com/google/android/ground/model/geometry/Coordinate.kt

```diff
@@ -19,6 +19,7 @@ import kotlinx.serialization.Serializable
 
 /** A coordinate on a two-dimensional Cartesian plane. */
 @Serializable
-data class Coordinate(val x: Double, val y: Double) : Comparable<Coordinate> {
-  override fun compareTo(other: Coordinate): Int = compareValuesBy(this, other, { it.x }, { it.y })
+data class Coordinate(val latitude: Double, val longitude: Double) : Comparable<Coordinate> {
```

Comment:
I would update the doc comment as well to state that we explicitly mean a coordinate in ref to the Earth and using https://en.wikipedia.org/wiki/Decimal_degrees to represent lat/lng (note that originally this was just a plain Real-numbered cartesian coordinate, so the concept is different (and better!) now).